mnd/mnd/registry/patients/mnd_admin_forms.py

```diff
@@ -284,3 +287,20 @@ def _clean_fields(self):
             for f in required_fields:
                 self.fields[f].required = True
         super()._clean_fields()
+
+
+class DuplicatePatientForm(PrefixedModelForm):
+    is_duplicate = forms.BooleanField(label=_("Mark this patient as a potential duplicate"), required=False)
+
+    class Meta:
+        model = DuplicatePatient
+        fields = ('is_duplicate',)
+
+    def save(self, commit=True):
```

Comment:
Is it better to use signals here?

Reply:
If this is the single place when the notification is triggered then probably not.